src/main/resources/view/ContactDetailCard.fxml

```diff
@@ -11,65 +11,59 @@
 <?import javafx.scene.text.Font?>
 
 <HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-    <GridPane HBox.hgrow="ALWAYS">
-        <columnConstraints>
-            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
-        </columnConstraints>
-        <VBox alignment="CENTER_LEFT" minHeight="105" spacing="10" GridPane.columnIndex="0">
-            <padding>
-                <Insets bottom="5" left="15" right="5" top="5" />
-            </padding>
-
-            <HBox alignment="CENTER_LEFT" spacing="10">
-                <padding>
-                    <Insets bottom="10" />
-                </padding>
-                <Label fx:id="nameView" minHeight="-Infinity" minWidth="-Infinity" prefHeight="56.0" prefWidth="280.0" style="-fx-text-fill: black; -fx-font-size: 30; -fx-background-color: transparent;" styleClass="cell_view_title_label" text="\$first">
+<GridPane maxWidth="-Infinity" style="-fx-background-color: transparent;">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" />
+    </columnConstraints>
+    <VBox alignment="CENTER_LEFT" minHeight="105" spacing="10" GridPane.columnIndex="0">
+        <padding>
+            <Insets bottom="5" left="15" right="5" top="5" />
+        </padding>
+         <HBox VBox.vgrow="ALWAYS">
+            <children>
+               <Label fx:id="nameView" style="-fx-text-fill: black; -fx-font-size: 30; -fx-background-color: transparent;" text="\$first" wrapText="true">
```

Comment:
would it be better to move the `nameView` outside of the ScrollPane so that the name will always be displayed when the user scrolls down

Reply:
good idea 👍